variables.tf

```diff
@@ -189,13 +189,13 @@ variable "kubeconfig_name" {
 variable "cluster_create_timeout" {
   description = "Timeout value when creating the EKS cluster."
   type        = string
-  default     = "15m"
+  default     = "30m"
 }
 
 variable "cluster_delete_timeout" {
   description = "Timeout value when deleting the EKS cluster."
   type        = string
-  default     = "15m"
+  default     = "30m"
```

Comment:
It sounds like the default for delete timeout is `15m` https://www.terraform.io/docs/providers/aws/r/eks_cluster.html#timeouts

Reply:
![no idea](https://media.giphy.com/media/bPTXcJiIzzWz6/source.gif)

I've changed and squashed